cmake/onnx/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # Licensed under the MIT License.
 
 add_library(onnx_proto ${ONNXRUNTIME_ROOT}/core/protobuf/onnx-ml.proto ${ONNXRUNTIME_ROOT}/core/protobuf/onnx-operators-ml.proto)
-target_include_directories(onnx_proto PUBLIC $<TARGET_PROPERTY:protobuf::libprotobuf,INTERFACE_INCLUDE_DIRECTORIES> "${CMAKE_CURRENT_BINARY_DIR}/..")
+target_include_directories(onnx_proto PUBLIC "${CMAKE_CURRENT_BINARY_DIR}/..")
```

Comment:
Why removing $<TARGET_PROPERTY:protobuf::libprotobuf,INTERFACE_INCLUDE_DIRECTORIES> ?
onnx_proto should depends on google protobuf header files.